pkg/kritis/cryptolib/jwt.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cryptolib
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"github.com/pkg/errors"
+)
+
+func decodeBase64(in []byte) ([]byte, error) {
```

Comment:
Can you document this function in comments?

Reply:
I removed this function as base64.RAWURLEncoding handles decoding without padding.